<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBacklight</class>
 <widget class="QWidget" name="FormBacklight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>1280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>1280</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
font: 25px &quot;微软雅黑&quot;;
 background-color:rgb(0, 0, 0);
}

QLabel
{
    color: rgb(255, 255, 255);
}
QLabel#label_curCyc ,QLabel#label_curInMold ,QLabel#label_lastSumMold
{
	font:70px &quot;方正舒体&quot;;
}
QTimeEdit
{
	font:70px &quot;微软雅黑&quot;;
	background-color:rgb(0, 0, 0);
border: 2px solid rgb(255,255,255);
}

QDateEdit
{
font: 35px &quot;方正舒体&quot;;
background-color:rgb(0, 0, 0);
border: 2px solid rgb(255,255,255);
/*border: 2px solid qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 red, stop:0.5 green, stop:1 blue);		/*2px粗，实线，渐变*/
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frameBacklight">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>335</x>
        <y>120</y>
        <width>110</width>
        <height>34</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>日期时间</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_curInMold">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>650</y>
        <width>281</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>方正舒体</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>20 s</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>200</width>
        <height>70</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>方正舒体</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_lastSumMold">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>850</y>
        <width>281</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>方正舒体</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>5000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>335</x>
        <y>410</y>
        <width>110</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>循环周期</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>335</x>
        <y>800</y>
        <width>110</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>生产模数</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>335</x>
        <y>603</y>
        <width>110</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>取件时间</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>351</width>
        <height>121</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>5</hour>
        <minute>54</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>HH:mm:ss</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_curCyc">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>460</y>
        <width>281</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>方正舒体</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>20 s</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
